lib/Twig/Profiler/Profile.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) 2015 Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+class Twig_Profiler_Profile implements \IteratorAggregate, \Serializable
```

Comment:
this breaks the PHP 5.2 compat (see the Travis failures). You should remove the ``\``